src/components/Table/index.js

```diff
@@ -0,0 +1,261 @@
+// @flow
+import React, { type Node } from 'react';
+import styled from 'styled-components';
+import colors from 'config/colors';
+import { uiColors, fontSizes, fontWeights } from 'utils';
+import Checkbox from 'components/Checkbox';
+import Icon from 'elements/Icon';
+
+const TableWrapper = styled.table`
+  background: ${colors.white};
+  width: 100%;
+  text-align: left;
+  border-collapse: collapse;
+  color: ${uiColors('text.light')};
+  box-sizing: border-box;
+  border: 1px solid ${colors.gray};
+  border-radius: 3px;
+  position: relative;
+  overflow-x: hidden;
+  font-weight: ${fontWeights('light')};
+
+  .entryname {
+    color: ${uiColors('text.main')};
+  }
+
+  .checkbox {
```

Comment:
Instead of adding more classes, we can use styled-component's component selectors, i.e we can use the `Checkbox` component directly to select it:
```suggestion
  ${Checkbox} {
```

This will select all Checkbox components that are rendered as children of TableWrapper.